pkg/controller/woopra_test.go

```diff
@@ -73,7 +73,7 @@ func (s *TestWoopraSuite) TestWoopraHandler() {
 		req, err := http.NewRequest(http.MethodGet, "/api/v1/segment-write-key", nil)
 		require.NoError(s.T(), err)
 
-		handler := gin.HandlerFunc(woopraCtrl.GetSegmentWriteKey)
+		handler := gin.HandlerFunc(woopraCtrl.GetSandboxSegmentWriteKey)
```

Comment:
I guess we should add tests for GetDevSpacesSegmentWriteKey too?

Reply:
done in [3befcfd](https://github.com/codeready-toolchain/registration-service/pull/323/commits/3befcfd9dcd0c7ea9c2515921c69413c5ef1f927)